minitwit/src/main/java/WebApplication.java

```diff
@@ -85,22 +89,147 @@ public static String render(Map<String, Object> model, String templatePath) {
     }
 
     public static Route servePublicTimelinePage = (Request request, Response response) -> {
+        try {
```

Comment:
Maybe its not something we should worry about now, but I have seen that basically all our functions just have a giant try catch over every line of code, seems a bit sketchy

Reply:
I agree, but without it all exceptions are caught by a generic code 500 page that doesn't tell anything about what happened anywhere. Maybe we could make some sort of wrapper around the server functions so its the same...